ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master.yaml

```diff
@@ -0,0 +1,52 @@
+base_images:
+  openshift_release_golang-1.15:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+images:
+- dockerfile_path: Dockerfile.assisted-installer
+  from: src
+  inputs:
+    openshift_release_golang-1.15:
+      as:
+      - registry.svc.ci.openshift.org/openshift/release:golang-1.15
+  to: assisted-installer
+promotion:
+  name: "4.7"
+  namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
+# test_binary_build_commands: |
+#   make build-openshift-ci-test-bin
```

Comment:
I think you should build the `Dockerfile.assisted-installer-build` somehow and run on top of it.
Because `skipper` was meant to be used both by developers and CI.